ProjetSupaero2018/README.md

```diff
@@ -1,6 +1,33 @@
 # Installation
-##ACADO
-TODO
+
+ROS Kinetic (full install) is required. See http://wiki.ros.org/ROS/Installation
+For example for Ubuntu, that would be:
+
+```
+sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
+sudo apt-get update
+sudo apt-get install ros-kinetic-desktop-full
+```
+
+ACADO Toolkit is also required.
+
+```
+sudo apt-get install gcc g++ cmake git gnuplot doxygen graphviz
+
+git clone https://github.com/acado/acado.git -b stable ACADOtoolkit && cd ACADOtoolkit
+git reset --hard 88c441b6bedee039ef8cb81d34fcd9377fb6d138
+mkdir build && cd build
+cmake .. && make
+sudo make install
+```
+
+### Add the models directory to your Gazebo model path
```

Comment:
Actually this might not be necessary any longer - a script does the trick when roslaunching.
It may be kept as is though, but has to be executed after compiling our project (instruction : "rospack find display" requires catkin_making our project)